projects/angular/src/utils/i18n/common-strings.default.ts

```diff
@@ -133,4 +133,7 @@ export const commonStringsDefault: ClrCommonStrings = {
   // Tree
   selectedTreeNode: 'selected',
   unselectedTreeNode: 'unselected',
+
+  // Breadcrumbs
+  breadcrumb: 'breadcrumb',
```

Comment:
```suggestion
  breadcrumbsLabel: 'breadcrumb's,
```

I think the property name should be more specific that this string is for a label, and I think the label should be plural.